library/src/include/logging.h

```diff
@@ -56,6 +56,17 @@ class tuple_helper
         }
     }
 
+    // Complex output
```

Comment:
We need to make sure we are handling log_bench correctly. Some options I think are:
- Handle alpha & beta test inputs as strings, convert to real or complex values afterwards, we could then simply log the values in a manner similar to this. (I believe this is how it's done in the complex gemm PR)
- Exceptionally handle log_bench logging for complex values to output alpha/beta and alphai/betai values. (This can be seen currently in rocblas_axpy.cpp, for example)